lib/torrent.js

```diff
@@ -549,7 +551,7 @@ export default class Torrent extends EventEmitter {
       })
     } else {
       // start off selecting the entire torrent with low priority
-      if (this.pieces.length !== 0) {
+      if (this.pieces.length !== 0 && !this._startAsDeselected) {
         this.select(0, this.pieces.length - 1, false)
```

Comment:
what about this, why false?

Reply:
I didn't change this line, but I guess in this case I could replace the `false` to `0`. They're basically the same thing here.
Replaced in last commit